src/main/java/carpet/script/CarpetExpression.java

```diff
@@ -2899,6 +2900,37 @@ private void API_AuxiliaryAspects()
             return (cc, tt) -> time;
         });
 
+        this.expr.addLazyFunction("unix_time",0,(c,t,lv)->{
+            Value time = new NumericValue(System.currentTimeMillis());
+            return (_c,_t)-> time;
+        });
+
+        this.expr.addLazyFunction("unix_date",0,(c,t,lv)->{
+            String[] list_date = new Date(System.currentTimeMillis()).toString().split(" ");
```

Comment:
dont use to string use Date and extract each components using its API
https://docs.oracle.com/javase/7/docs/api/java/util/Date.html#Date(long)

Reply:
Also don't return any strings if not needed. 
I suggest l(year, month, day, hour, minute, second, week_number, week_day)
also not sure how timezone is returned with Date - you might include that
so date(1234567) -> l(2020, 6, 1, 16, 30, 10, 19, 5) - everyone would format it as they please

it would be good to have a reverse operation as well so date(l(2020, 6, 1, 16, 30, 10)) -> 1234567
I am pretty sure Date class allows the reverse operation.

time numbers were made up for illustraiton.

Reply:
Idk how to get a list as an arg, so Ill make it temporarily have multiple args until I figure out how to do

Reply:
resolved